app/admin/age_range_options.rb

```diff
@@ -0,0 +1,36 @@
+ActiveAdmin.register AgeRangeOption do
+  permit_params :name, :spanish_name
```

Comment:
Why have a separate "name" and "spanish name"?

Isn't there a plan to eventually include other languages on the app?